src/solv/model.f90

```diff
@@ -210,48 +234,149 @@ subroutine loadInternalParam(self, env, solvent, level)
 
    type(gbsa_parameter), allocatable :: param
 
-   if (any(level == [2, 0])) then
-      self%paramFile = 'internal GFN2-xTB/GBSA'
-      select case(solvent)
-      case('acetone');      param = gfn2_acetone
-      case('acetonitrile'); param = gfn2_acetonitrile
-      case('benzene');      param = gfn2_benzene
-      case('ch2cl2','dichlormethane'); param = gfn2_ch2cl2
-      case('chcl3','chloroform');      param = gfn2_chcl3
-      case('cs2');          param = gfn2_cs2
-      case('dmso');         param = gfn2_dmso
-      case('ether');        param = gfn2_ether
-      case('h2o','water');  param = gfn2_h2o
-      case('methanol');     param = gfn2_methanol
-      case('thf');          param = gfn2_thf
-      case('toluene');      param = gfn2_toluene
-      case('dmf');          param = gfn2_dmf
-      case('nhexan','n-hexan','nhexane','n-hexane');
-         param = gfn2_nhexan
-      end select
-   else
-      self%paramFile = 'internal GFN1-xTB/GBSA'
-      select case(solvent)
-      case('acetone');      param = gfn1_acetone
-      case('acetonitrile'); param = gfn1_acetonitrile
-      case('benzene');      param = gfn1_benzene
-      case('ch2cl2','dichlormethane'); param = gfn1_ch2cl2
-      case('chcl3','chloroform');      param = gfn1_chcl3
-      case('cs2');          param = gfn1_cs2
-      case('dmso');         param = gfn1_dmso
-      case('ether');        param = gfn1_ether
-      case('h2o','water');  param = gfn1_h2o
-      case('methanol');     param = gfn1_methanol
-      case('thf');          param = gfn1_thf
-      case('toluene');      param = gfn1_toluene
-      end select
-   end if
-
-   if (.not.allocated(param)) then
-      call env%error("solvent: '"//solvent//"' is not parametrized", source)
-      return
-   end if
+   select case(self%kernel)
 
+  case (gbKernel%still)
```

Comment:
Please fix the indentation here, one indentation level is three spaces here, the block inside the `case` should be indented twice (six spaces).